Detectors/TOF/simulation/src/Digitizer.cxx

```diff
@@ -91,12 +95,21 @@ void Digitizer::init()
 
 int Digitizer::process(const std::vector<HitType>* hits, std::vector<Digit>* digits)
 {
+  static constexpr double MAX_HIT_TIME = 1000.; // hit time cutoff (slow neutrons) = 1 microsecond
```

Comment:
Should we put it in TOFSimParams.h configurable parameters?

Reply:
Can do later. For now I wouldn't change this.

Reply:
ok